### PR TITLE
Updated install.rst to be valid for Symfony3

### PR DESCRIPTION
**Pull request by dangreenisrael**
I am targetting this branch, because the documentation should be valid for the current version of Symfony
## Changelog

``` markdown

### Changed
- The `config.yml` `twig` section to be up to date with Symfony3

```
## Subject

Changed the `form` twig section to be valid for Symfony3

### CODE REVIEW

File: Resources/doc/reference/installation.rst [hunk 1]
```diff
@@ -42,9 +42,15 @@ Edit the ``config.yml`` file and add these lines:
         strict_variables: "%kernel.debug%"
 
         #sonata
-        form:
-            resources:
-                - 'SonataFormatterBundle:Form:formatter.html.twig'
+        
+        #Uncomment the following 3 liness for Symfony2
```
Comment (greg0ire) [inline Resources/doc/reference/installation.rst]: - liness => lines
- one space after each `#`, please

**Comment by greg0ire** [comment]
If this configuration step is mandatory, we might as well configure the twig bundle automatically thanks to [this trick](http://symfony.com/doc/current/bundles/prepend_extension.html). Your PR is still valid until then, of course.

**Comment by hug963** [comment]
Actually, the new config is valid since symfony 2.6:
https://github.com/symfony/symfony/blob/2.6/src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php#L57

File: Resources/doc/reference/installation.rst (hunk 1 above)
Comment (core23) [inline Resources/doc/reference/installation.rst]: Please fix @dangreenisrael

**Comment by greg0ire** [comment]
ping @dangreenisrael

**Comment by dangreenisrael** [comment]
Sorry guys, since I opened this thread I've taken a new job and we don't use PHP. So, I'm out of the Symfony game for now.

**Comment by greg0ire** [comment]
Congrats on your new job! Sorry it prevents you to finish this…

**Pull request closed without merging**